### PR TITLE
 Textual updates, clarifications and references added

### PR DESCRIPTION
**Pull request by Morrizzzzz**
Looking through this episode I added some textual edits and links.  I hope these are useful. 

1. In the section *Split data into training and test set*
- Link to the scikit-learn module added
- more clarification added on datasets (i.e. penguins_features) and target (i.e. defined as target).

2. In section, 4. *Build an architecture from scratch or choose a pretrained model* 
- Rephrased the first sentence

3. In section 5 *Choose a loss function and optimizer*
- Reference added to the Adam optimizer.

4. In section 8, *Measuring performance*:
- added an explicit refernce to the test set. 

5. In section 9, *Tune hyperparameters*
- Added: Note that the outcome you have might be slightly different from what is shown in this tutorial.

### CODE REVIEW

File: episodes/2-keras.Rmd [hunk 1]
```diff
@@ -336,8 +336,8 @@ either. So we will leave it as it is.
 ## 4. Build an architecture from scratch or choose a pretrained model
 
 ### Keras for neural networks
-For this lesson we will be using [Keras](https://keras.io/) to define and train our neural network
-models.
+We will now build out first neural network from scratch. Although this sounds like a daunting task, you will experience that with [Keras](https://keras.io/) it is actually surprisingly straightforward.
```
Comment (svenvanderburg) [inline episodes/2-keras.Rmd:339]:
```suggestion
We will now build our first neural network from scratch. Although this sounds like a daunting task, you will experience that with [Keras](https://keras.io/) it is actually surprisingly straightforward.
```

**Review by svenvanderburg — APPROVED** [review]
Great additions @Morrizzzzz, this is really useful. I fixed one small typo
